LaTexMD/index.md

```diff
@@ -0,0 +1,16 @@
+# Using LaTex in markdown
+
+Since GitHub markdown doesn't natively support LaTex, we use markdeep.  
```

Comment:
[markdeep](https://casual-effects.com/markdeep/)

Reply:
ah great, review sucks here, i did only select markdeep in `we use markdeep`

Reply:
I'm not sure what you're saying here. Can you rephrase?

Reply:
I wanted the word `markdeep` itself to be a link already

Reply:
Oh, sure, I'll fix that in a bit.